### PR TITLE
feat: delete space

### PR DESCRIPTION
**Pull request by j0hnfl0w**
Fixes https://github.com/snapshot-labs/snapshot/issues/3580

## Changes
- removed primary prop on cancel button in ModalConfirmAction

## How to test
- try to delete space
- create space again and delete

### CODE REVIEW

**Comment by samuveth** [comment]
1. We should add an input field to the confirmation modal where the user has to type the space ID that he is deleting. Also 
2. After deleting the space the spaces should be reloaded, currently if I go back to that space it still shows.

3. Why do I get redirected to a user profile instead of the homepage after deleting the space.

4. How are we doing to handle this on custom domains?

**Comment by j0hnfl0w** [comment]
1 ok, it would be great if it was in the issue from the start :D
2 also we can just reload a page?
3 why not? thats where you can see your spaces.
4 just like we handle viewProfile action in MenuAccount

**Comment by samuveth** [comment]
1. Yeah we discussed it on the call 2 weeks ago, forgot to add it
2. We can add a new method here to remove the space from the object
3. Would feel more natural to go to the home page and if on custom domain go to snapshot.org homepage

**Comment by samuveth** [comment]
After deleting one of my spaces it remains in the sidebar. We should remove it from `extendedSpaces` after deletion

File: .env [hunk 1]
```diff
@@ -2,7 +2,7 @@ VITE_HUB_URL=https://testnet.snapshot.org
 VITE_RELAYER_URL=https://testnet.snapshot.org
 VITE_SCORES_URL=https://score.snapshot.org
 VITE_IPFS_GATEWAY=snapshot.mypinata.cloud
-VITE_DEFAULT_NETWORK=1
+VITE_DEFAULT_NETWORK=5
```
Comment (j0hnfl0w) [inline .env:5]: @samuveth wdut maybe we can set default network to goerli? we are in testnet anyway

File: src/components/ModalConfirmAction.vue [hunk 2]
```diff
@@ -19,17 +20,13 @@ defineEmits(['close', 'confirm']);
 
     <template #footer>
       <div class="flex gap-3">
-        <BaseButton
-          v-if="showCancel"
-          class="w-full"
-          primary
```
Comment (j0hnfl0w) [inline src/components/ModalConfirmAction.vue:25]: remove primary prop from cancel button cos buttons should not be the same

File: .env (hunk 1 above)
Comment (samuveth) [reply .env:5]: For that we will have to change some ENV on fleek and vercel. Now if you open the preview deployment we are on `hub.snapshot.org` but the network is on Goerli.

Comment (samuveth) [reply .env:5]: We should handle this in anther PR/Issue

Comment (samuveth) [reply .env]: https://github.com/snapshot-labs/snapshot/issues/3705

**Comment by samuveth** [comment]
Great work @j0hnfl0w

**Pull request merged**